base/utils/R/utils.R

```diff
@@ -324,7 +324,7 @@ pdf.stats <- function(distn, A, B) {
     norm = B ^ 2,
     f = ifelse(B > 4,
                2 * B^2 * (A + B - 2) / (A * (B - 2) ^ 2 * (B - 4)),
-               stats::var(stats::rf(1e+05, A, B))))
+               ncdf4::nc_var(stats::rf(1e+05, A, B))))
```

Comment:
This is a very different usage than your previous changes. This is the function `stats::var` which calculates variance. This one should be reverted to it's previous text

Reply:
understood